From 126cc473b8455205e8ee2d6b1aff06de0d2b681f Mon Sep 17 00:00:00 2001
From: Mike Snitzer <snitzer@redhat.com>
Date: Fri, 13 Jun 2014 14:34:51 -0400
Subject: [block] make __blkdev_issue_zeroout static

Message-id: <1402670238-13916-4-git-send-email-snitzer@redhat.com>
Patchwork-id: 83759
O-Subject: [RHEL7.1 PATCH 003/150] block: make __blkdev_issue_zeroout static
Bugzilla: 1105204
RH-Acked-by: David Milburn <dmilburn@redhat.com>
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

BZ: 1105204

Upstream commit 35086784caec571be185f643eb1b045a275d60b3
Author: Fabian Frederick <fabf@skynet.be>
Date:   Mon May 26 22:19:14 2014 +0200

    block/blk-lib.c: make __blkdev_issue_zeroout static

    __blkdev_issue_zeroout is only used in blk-lib.c

    Cc: Jens Axboe <axboe@kernel.dk>
    Cc: Andrew Morton <akpm@linux-foundation.org>
    Signed-off-by: Fabian Frederick <fabf@skynet.be>
    Signed-off-by: Jens Axboe <axboe@fb.com>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/block/blk-lib.c b/block/blk-lib.c
index 4cb9ee8..35cdebc 100644
--- a/block/blk-lib.c
+++ b/block/blk-lib.c
@@ -226,8 +226,8 @@ EXPORT_SYMBOL(blkdev_issue_write_same);
  *  Generate and issue number of bios with zerofiled pages.
  */
 
-int __blkdev_issue_zeroout(struct block_device *bdev, sector_t sector,
-   sector_t nr_sects, gfp_t gfp_mask)
+static int __blkdev_issue_zeroout(struct block_device *bdev, sector_t sector,
+      sector_t nr_sects, gfp_t gfp_mask)
 {
  int ret;
  struct bio *bio;
-- 
1.7.1